tests/test_user_management.py

```diff
@@ -25,7 +31,27 @@ def test_success(self):
 
         self.assertCountEqual(actual.keys(), expected_keys)
 
-    def test_failure(self):
+    def test_user_by_token_failure(self):
         with self.assertRaises(requests.exceptions.HTTPError):
             _ = get_user_information('https://cloud-backend.lanalabs.com',
                                      token='not-a-valid-token ')
+
+    def test_list_users(self):
+        users = self.api.get_all_users()
+        self.assertGreaterEqual(len(users), 1)
+
+    def test_user_management(self):
+        resp_user_info = self.api.get_user_information_by_id('e44e8b55-1b28-4db7-9396-6acd4e94ee2a')
+
+        self.assertEqual(resp_user_info['role'], 'UserAdmin')
```

Comment:
WIth the above setup the expected role here should match the role of the api used in the previous line

```python
self.assertEqual(resp_user_info['role], self.api_r_analyst.user.role)
```

we use differengt code in the function to obtain the same information, so the code checks each other.